crates/sui-framework/packages/sui-system/sources/voting_power.move

```diff
@@ -13,11 +13,18 @@ module sui_system::voting_power {
     #[test_only]
     friend sui_system::voting_power_tests;
 
+    /// Deprecated. Use VotingPowerInfoV2 instead.
     struct VotingPowerInfo has drop {
         validator_index: u64,
         voting_power: u64,
     }
 
+    struct VotingPowerInfoV2 has drop {
+        validator_index: u64,
+        voting_power: u64,
+        stake: u64,
+    }
+
```

Comment:
Since this is a private type do we actually need to do this?

Reply:
Unfortunately yes :/ We don't have a notion of private type in Move right now.